src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,130 @@
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: $card-gap-vertical $card-gap-horizontal;
+  padding: $catalog-padding-vertical $catalog-padding-horizontal;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr); // 2 columns starting from 488px
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 1fr); // 3 columns starting from 768px
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 1fr); // 4 columns starting from 1024px
+  }
+}
+
+.card {
+  background: $color-bg-white;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
```

Comment:
The `border-radius` property is defined twice for the `.card` class. Please remove the duplicate declaration to ensure consistency and avoid confusion.